### ISSUE TITLE
Over-segmentation

### ISSUE THREAD

**Issue by WeheliyeHashi** [kevinjohncutler/omnipose#5]
Dear Omnipose,

Many thanks for your code. I have managed to train your code on the segmentation of c.elegan worms. Please see how nicely the code segments it. However, I just have one small question regarding the over-segmentation. Everytime, I evaluate the model I ensure the omni is set to true as shown in the code below:

masks, flows, styles = model.eval([imgs[i] for i in tqdm.tqdm(n)],channels=chans,dist_threshold=dist_threshold,
                                  flow_threshold=0,resample=True,
                                  omni = False, normalize=True, verbose=verbose,diam_threshold=diam_threshold,
                                  cluster=cluster,rescale=None, diameter =15)

Whenever I set the omni to False I get the over-segmentation as shown in Figure_6_Omni_False (see attached figure). Following your paper to get rid of the over-segmentation of Figure_6_Omni_False, I tried the following lines of code but I was unsuccessful in getting rid of the over-segmentation. Is there something that I am missing. I really want to get it to be the same as Figure_6_omni_True.
masked_image = Figure_6_Omni_False
labels, dists, heat, flows = masks_to_flows(masked_image, use_gpu=True, omni=True)
plt.imshow(plot.dx_to_circ(flows, transparency=True)) 
masks = masked_image
dP = flows
import edt
dists = edt.edt(masks)
dists[dists==0]=-5
inds = np.array(np.nonzero(masks)).astype(np.int32).T
print(inds.shape)
mask_pred, p, tr = compute_masks(dP,dists,inds=inds,calc_trace=True,use_gpu=0,
                                           omni=True,niter=500,interp=True,mask_threshold=-1,verbose=True,
                                          flow_threshold = 0.0,min_size=0, cluster=True)

![Figure_6_omni_True](https://user-images.githubusercontent.com/88719125/162977049-20f52008-5c24-4465-8410-3eed15d534f2.png)
![Figure_6_omni_False](https://user-images.githubusercontent.com/88719125/162977812-e8b1743f-2dba-439a-8387-1295addb7f4b.png)
![Figure_6_over_segmentation](https://user-images.githubusercontent.com/88719125/162977891-3d05cc31-4c24-4502-b7df-2ae8afda83eb.png)
